packages/layout/tests/node/setAlignSelf.test.js

```diff
@@ -1,7 +1,12 @@
-import Yoga from 'yoga-layout/sync';
+import { jest } from '@jest/globals';
+import yogaModule from 'yoga-layout/sync';
 
 import setAlignSelf from '../../src/node/setAlignSelf';
 
+// yoga-layout sets default export using non-standard __esModule property, so we need to
+// make an additional check in case it's used in a bundler that does not support it.
+const Yoga = 'default' in yogaModule ? yogaModule.default : yogaModule;
```

Comment:
Can we extract this in a `layout/yoga.js` file? We can capture it and I can do it separately. It might not be necessary either as I'd like to experiment with async yoga which I think it's more performant and uses wasm build

Reply:
Sure, defining that 35 times was idiotic :D Fixed that.

I don't think move to wasm will change anything here though.